ui/src/alerts/components/AlertsTableRow.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {PureComponent} from 'react'
+import {Link} from 'react-router'
+
+import classnames from 'classnames'
+
+import {ALERTS_TABLE} from 'src/alerts/constants/tableSizing'
+
+interface Props {
+  sourceID: string
+  name: string | null
```

Comment:
does the type need `string | null` or can this just be `null`?

Reply:
it appears that any of these values can end up as null if they aren't set from the tickscript